### ISSUE TITLE
Problem non JSON message (Xiaomi 2ch Relay)

### ISSUE THREAD

**Issue by ciotlosm** [yllibed/Zigbee2MqttAssistant#92]
Many features break if you set names with special characters. 

```
      Error processing MQTT message on topic 'zigbee2mqtt/Climă parter 2/l2/set'
Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: O. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonTextReader.Read()
   at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader, JsonLoadSettings settings)
   at Newtonsoft.Json.Linq.JObject.Parse(String json, JsonLoadSettings settings)
   at Zigbee2MqttAssistant.Services.BridgeStateService.UpdateDevice(String friendlyName, String jsonPayload, Boolean& forceLastSeen) in d:\a\1\s\Zigbee2MqttAssistant\Services\BridgeStateService.cs:line 59
   at Zigbee2MqttAssistant.Services.MqttConnectionService.DispatchZigbee2MqttMessage(MqttApplicationMessage msg) in d:\a\1\s\Zigbee2MqttAssistant\Services\MqttConnectionService.cs:line 404
   at Zigbee2MqttAssistant.Services.MqttConnectionService.HandleApplicationMessageReceivedAsync(MqttApplicationMessageReceivedEventArgs eventArgs) in d:\a\1\s\Zigbee2MqttAssistant\Services\MqttConnectionService.cs:line 328
```

I can't generate anymore the network map.

**Comment by carldebilly** [comment]
I tried with a lot of special charaters ... even emojis and it was working.

I just tried with some the charaters you just set...
![image](https://user-images.githubusercontent.com/4174207/67643660-1cc4a680-f8f0-11e9-9bfe-8aeea7fb7a76.png)

Maybe it's related to the slashes in the name?

Carl.

**Comment by ciotlosm** [comment]
The list of devices is fine, even renaming. The problem appears when I try to scan to get the network map.

**Comment by carldebilly** [comment]
Please give me more details about the problem you are experiencing...
* Docker version (cpu type & OS)
* Installation type (docker or HASS.IO)
* Versions of Z2M + Z2MA
* Steps to reproduce
* Pertinent logs
* If you can, a dumb of the problematic message on `zigbee2mqtt/Climă parter 2/l2/set` (you can use MQTT explorer or any other MQTT client app)

**Comment by ciotlosm** [comment]
Docker version (cpu type & OS) 
`I'm using RPI 3B with HassOS 3.5`

Installation type (docker or HASS.IO)
`HassOS 3.5`

Versions of Z2M + Z2MA
```
Version of Zigbee2MqttAssistant (this software): 0.3.89 (0.3.89+Branch.master.Sha.d881022fb4928b73f3d2ed27ba0927ca51178c9c) Release
Version of Zigbee2Mqtt: 1.6.0
Coordinator version: 20190619
Coordinator Zigbee Id:
```

Steps to reproduce

1. Go to Z2MA
2. Go to Map tab
3. Press Launch New Network Scan
4. Wait for scan to finish  (wait for a bunch of LQI errors in Z2M)
```
zigbee2mqtt:error 2019-10-28T17:25:51: Failed to execute LQI for 'Mansardă lumină'
zigbee2mqtt:error 2019-10-28T17:26:01: Failed to execute LQI for 'Baie etaj - lumină'
zigbee2mqtt:error 2019-10-28T17:26:11: Failed to execute LQI for 'Dormitor lumină'
zigbee2mqtt:error 2019-10-28T17:26:21: Failed to execute LQI for 'Dormitor lumină 1'
zigbee2mqtt:error 2019-10-28T17:26:31: Failed to execute LQI for 'Dormitor lumină 2'
zigbee2mqtt:error 2019-10-28T17:26:41: Failed to execute LQI for 'Dressing lumină'
zigbee2mqtt:error 2019-10-28T17:26:52: Failed to execute LQI for 'Scări - lumină parter'
zigbee2mqtt:error 2019-10-28T17:27:02: Failed to execute LQI for 'Curte lumină'
```
5. Errors like the above in Z2MA logs


> Pertinent logs
Unfortunately I restarted the addon again to start fresh, and now it worked and don't have older logs. Maybe it's related to some memory problems. I did update the addon today (Z2MA) to latest release. 

> If you can, a dumb of the problematic message on zigbee2mqtt/Climă parter 2/l2/set (you can use MQTT explorer or any other MQTT client app)
If I can reproduce again, I'll post here, if I can't in a day or two, I'll just assume latest update somehow solved this.

**Comment by carldebilly** [comment]
I just found the problem...  It's at this line:
<https://github.com/yllibed/Zigbee2MqttAssistant/blob/master/Zigbee2MqttAssistant/Services/MqttConnectionService.cs#L58>

It was there before being able to put slashes in the name of the device and this scenario hasn't been tested.

It's generating an error in the log, but it's not suppose to have any impact on the operations of the software...

Let me check if I can fix it quickly...

**Comment by carldebilly** [comment]
@all-contributors please add ciotlosm to bugs

**Comment by carldebilly** [comment]
Fixed in v`0.3.90`! Please try it.
